content/_layouts/redirect.html.haml

```diff
@@ -0,0 +1,37 @@
+- # Define a redirect elsewhere with optional support for browser language detection
+- # Configuration is via front matter, one URL is the non-JS default, others are redirected to via JS if language matches
+- #
+- # ---
+- # layout: redirect
+- # redirect_url: "https://wiki.jenkins-ci.org/display/JENKINS/Remote+access+API"
+- # localized_urls:
+- #   ja: "https://wiki.jenkins-ci.org/display/JA/Remote+access+API"
+- #   de: "https://wiki.jenkins-ci.org/display/DE/Remote+access+API"
+- # ---
+
+
+:ruby
+  single_url = if page.localized_urls then false else true end
+
+
+%html
+  %head
+    %meta{:'http-equiv' => 'content-type', :content => 'text/html; charset=utf-8'}/
+    - if single_url
+      %meta{:'http-equiv' => 'refresh', :content => "0;URL=#{page.redirect_url}"}/
+    - else
+      %noscript
+        %meta{:'http-equiv' => 'refresh', :content => "0;URL=#{page.redirect_url}"}/
+  %body
+    %center
+      This content has moved to
+      %a{:href => page.redirect_url}
+        = page.redirect_url
+    %script
+      var language = window.navigator.userLanguage || window.navigator.language;
```

Comment:
I know from unfortunate experience that `navigator.language` doesn't return the value of the `Accept-Language` header; that would have been too sensible.  Instead, it returns the language that the browser UI is displayed in, which may (or may not) be the operating system language.

There is apparently `navigator.languages` in newer Chrome/Firefox/Safari to return useful values, but I would guess that most Japanese developers have their computers set to Japanese anyway, so it's unlikely to be a big problem.  But I thought it was worth mentioning.